DiscordBot/config.py

```diff
@@ -0,0 +1,156 @@
+import os
+import random
+import sys
+from configparser import ConfigParser
+
+__config = ConfigParser()
```

Comment:
I'm surprised you didn't just use json lol

Reply:
.ini config is more convenient and compact for regular users and there is already a built-in library for serialization of config